### DISCUSSION TITLE
Take care about MojoHaus

### DISCUSSION THREAD

**Discussion by slawekjaranowski** [support-and-care/maven-support-and-care#26] [Ideas for Funding]
There are a many useful and frequently used plugins under https://github.com/mojohaus

Eg build-helper-maven-plugin, exec-maven-plugin, versions-maven-plugin

The best way will be donated to ASF, even not happen - we should take care about maintenance and quality of them.

**Comment by hendrikebbers** [comment]
@khmarbaise who is the "owner" of Mojohaus? Who need to agree on such movement of the repos?

**Comment by khmarbaise** [comment]
To "donate" plugins to the ASF you must go through the IP clearance process (https://incubator.apache.org/guides/ip_clearance.html)  that means all contributors (for each plugin) have to be asked for their approval etc. (to apply their changes in accordance with the Apache License).. also some of the plugins do not have an Apache License which could be hard if not even impossible .... for example the build-helper-maven-plugin has a MIT License which means a full change of the license... (Is there some advocate around which could be asked for that; It might be an option to ask ASF itself for that)...

**Comment by olamy** [comment]
TBH donation to ASF would is a bit complicated and without any real added value.
 I'd rather prefer to have only core plugins at ASF.
